pkg/operator/staticresource/controller.go

```diff
@@ -0,0 +1,289 @@
+package staticresource
```

Comment:
Could I request a controller rename, this feels awspodidentity specific and not as generic as staticresource sounds. If we had additional static resource setup to do, we'd probably want that in another controller rather than forcing into this one.

Reply:
Yeah, I thought in the beginning I would make this more generic but... it didn't happen.  I can rename.